Modificacion al archivo de entrada que mencione en mi issue e implementacion de nuevo comando "ImprimirLn" para manejar impresiones con salto y sin salto

HPecene opened the pull request:
El codigo se ve muy ordenado y despues de arruinarlo (editarlo) no encontre algo mas por el momento.

HPecene committed:
Modificacion al archivo de entrada que mencione en mi issue

El codigo se ve muy ordenado y despues de arruinarlo (editarlo) no encontre algo mas por el momento.

rjanixz reviewed (changes requested):
El cambio de los IFs anidados no es necesario, es una demostración de este funcionamiento.

HPecene committed:
Update Gramatica.cs

Se agrego un nuevo simbolo, RIMPRIMIRLN a la gramatica para hacer una impresion continua sin salto.

HPecene committed:
Create ImprimirLn.cs

Se crea la clase que maneja el comando de Imprimir con Salto

HPecene committed:
Update Imprimir.cs

Se cambia el Write para manejar sin salto en "Imprimir"

HPecene committed:
Update ImprimirLn.cs

Se cambia el Write para agregar un salto de linea a "Imprimirln"

HPecene committed:
Update Sintactico.cs

Se agrego la accion en el caso de ImprimirLn

HPecene committed:
Update InterpreteSencillo.csproj

Se agrega ImprimirLn.cs a los archivos a compilar

HPecene committed:
Update entrada.txt

Cambiamos el archivo de entrada para incluir "imprimirln" donde sea conveniente un salto de linea y dejamos "imprimir" donde sea mejor solo imprimir en la misma linea

ericknavarro commented:
Muchas gracias por tu aporte José.

Saludos cordiales.
